pypeit/reduce.py

```diff
@@ -165,19 +166,19 @@ def _chk_objs(self, items):
 
```

Comment:
Please make the corresponding set of changes to the find_objects method in the Echelle child, and not just the multislit child that you are using for NIRSPEC.

Reply:
this is in the parent so no change needed.